uniffi_bindgen/src/bindings/mod.rs

```diff
@@ -133,3 +122,10 @@ pub fn write_bindings(
     }
     Ok(())
 }
+
+pub fn write_file(path: &Utf8Path, contents: String) -> Result<()> {
+    println!("{path}");
```

Comment:
This `println!()` was actually intended.  It makes it easier to debug issues if you can see what's being generated.  I can split that out into a separate PR if you have concerns.  Otherwise I can add a comment to make it more clear.